qa/run-tests.xml

```diff
@@ -537,12 +537,10 @@
             <then>
               <var name="jdbcstore.database.properties" value="ClassName=${jdbc.datasource.class};ServiceName=${jdbc.db.name};User=${jdbc.db.user};Password=${jdbc.db.password};ServerName=${jdbc.db.url};PortNumber=${jdbc.db.port}${jdbc.db.properties}" />
             </then>
-          <else>
-            <then>
+            <else>
               <var name="jdbcstore.database.properties" value="ClassName=${jdbc.datasource.class};DatabaseName=${jdbc.db.name};User=${jdbc.db.user};Password=${jdbc.db.password};ServerName=${jdbc.db.url};PortNumber=${jdbc.db.port}${jdbc.db.properties}" />
-            </then>
-          </else>
-          </if>
+            </else>
+	  </if>
```

Comment:
@jmfinelli I am not sure what is going on with these changes in here, please can you clarify?

Reply:
Oversight. `else` does not need `then`.

Reply:
Ah, so IIUC then the change in https://github.com/jbosstm/narayana/pull/1946/files did not work as it was expected to and the syntax is therefore incorrect as it stands?

I think this change should be in a separate commit but can be in the same PR

Reply:
> Ah, so IIUC then the change in https://github.com/jbosstm/narayana/pull/1946/files did not work as it was expected to and the syntax is therefore incorrect as it stands?

Yeah, that is what happened.

> I think this change should be in a separate commit but can be in the same PR

No problem, I will update this PR